internal/parser.go

```diff
@@ -111,13 +110,12 @@ func (p *Parser) parseTasks() error {
 		return err
 	}
 
-	re := regexp.MustCompile(osCommandRegexp)
-	allFilesPaths := []string{}
+	var allFilesPaths []string
 
 	for k, c := range tasks {
-		filePaths := []string{}
+		var filePaths []string
```

Comment:
The variable initialization to an empty array is deliberate, so that I don't have to check against nil when iterating over it.

Reply:
ok ok, i'll restore this